app.json

```diff
@@ -104,7 +104,7 @@
         "js_module": "endpoints/keyEndpoint.js",
         "js_function": "unwrapKey",
         "forwarding_required": "sometimes",
-        "authn_policies": ["jwt", "member_cert", "user_cert"],
+        "authn_policies": [],
```

Comment:
It is better to keep the jwt. The flexibility is in the validation policy where the coordinators can decide which properties are required to provide access to the key endpoints.

Reply:
Didn't we discussed this earlier, and suggestion was to remove it from app.json? I am flexible on approach as long as we don't have any additional process for Ad tech to onboard in alpha.